transports/transport-rest/transport-rest-vertx/src/test/java/org/apache/servicecomb/transport/rest/vertx/TestTransportConfig.java

```diff
@@ -63,174 +67,252 @@ int availableProcessors() {
 
   @Test
   public void testGetThreadCountNormal() {
-    ArchaiusUtils.setProperty("servicecomb.rest.server.thread-count", 10);
-    Assert.assertEquals(10, TransportConfig.getThreadCount());
+    try {
+      ArchaiusUtils.setProperty("servicecomb.rest.server.thread-count", 10);
+      Assert.assertEquals(10, TransportConfig.getThreadCount());
+    } finally {
+      ArchaiusUtils.setProperty("servicecomb.rest.server.thread-count", null);
+    }
```

Comment:
@After
  public void after() {
    ArchaiusUtils.resetConfig();
  }


May have done the job, do you find any problem this do not have this change?

Reply:
I don't know this method. You mean all our tests don't rely on `config`. They can be cleared?

Reply:
yes.